### PR TITLE
[onert] Use explicit copy constructor for Operand

### PR DESCRIPTION
**Pull request by hseok-oh**
Define ir::Operand copy contructor explicitly
Fix to reference operand if copy is needless

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Review by wateret — DISMISSED** [review]
LGTM

**Comment by wateret** [comment]
Please resolve conflicts.

File: runtime/onert/core/include/ir/Operand.h [hunk 1]
```diff
@@ -40,6 +40,7 @@ class Operand
   {
     // DO NOTHING
   }
+  explicit Operand(const Operand &) = default;
```
Comment (hyunsik-yoon) [inline runtime/onert/core/include/ir/Operand.h:43]: :100:

**Comment by hseok-oh** [comment]
@hyunsik-yoon @wateret I've rebased. PTAL

**Review by hyunsik-yoon — APPROVED** [review]
LGTM

**Review by wateret — APPROVED** [review]
LGTM

**Review by ragmani — APPROVED** [review]
LGTM

**Pull request merged**